doc/source/cluster/cluster_under_construction/running-applications-on-ray-clusters/monitoring-and-observability.rst

```diff
@@ -0,0 +1,184 @@
+3. :ref:`Prometheus metrics <multi-node-metrics>` for internal and custom user-defined metrics.
+
+For more information on these tools, check out the more comprehensive :ref:`Observability guide<observability>`.
+
+The rest of this page will focus on how to access these services when running a Ray Cluster.
+
+Monitoring the cluster via the dashboard
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+:ref:`The dashboard <ray-dashboard>` provides detailed information about the state of the cluster,
+including the running jobs, actors, workers, nodes, etc.
+By default, the :ref:`cluster launcher <ref-cluster-quick-start-vms-under-construction>` and :ref:`KubeRay operator <kuberay-quickstart>` will launch the dashboard, but will
+not publicly expose the port.
+
+.. tabbed:: If using the VM cluster launcher
+
+    You can securely port-forward local traffic to the dashboard via the ``ray
+    dashboard`` command.
+
+    .. code-block:: shell
+
+        $ ray dashboard [-p <port, 8265 by default>] <cluster config file>
+
+    The dashboard will now be visible at ``http://localhost:8265``.
+
+.. tabbed:: If using Kubernetes
+
+    The KubeRay operator makes the dashboard available via a Service targeting
+    the Ray head pod, named ``<RayCluster name>-head-svc``. You can access the
+    dashboard from within the Kubernetes cluster at ``http://<RayCluster name>-head-svc:8265``.
+
+    You can also view the dashboard from outside the Kubernetes cluster by
+    using port-forwarding:
+
+    .. code-block:: shell
+
+        $ kubectl port-forward service/raycluster-autoscaler-head-svc 8265:8265
+
+    For more information about configuring network access to a Ray cluster on
+    Kubernetes, see the :ref:`networking notes <kuberay-networking>`.
+
+
+Using Ray Cluster CLI tools
+^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Ray provides several CLI tools for observing the current cluster and
+application state.  The ``ray status`` command provides information about the
+current status of nodes in the cluster, as well as information about
+autoscaling. Other Ray CLI tools allow you to read logs across the cluster and
+summarize application state such as the currently running tasks and actors.
+These tools are summarized :ref:`here <state-api-overview-ref>`.
+
+These CLI commands can be run on any node in a Ray Cluster. Examples for
+executing these commands from a machine outside the Ray Cluster are provided
+below.
+
+.. tabbed:: If using the VM cluster launcher
+
+    Execute a command on the cluster using ``ray exec``:
+
+    .. code-block:: shell
+
+        $ ray exec <cluster config file> "ray status"
+
+.. tabbed:: If using Kubernetes
+
+    Execute a command on the cluster using ``kubectl`` and the configured
+    RayCluster name. We will use the Service targeting the Ray head pod to
+    execute a CLI command on the cluster.
+
+    .. code-block:: shell
+
+        $ kubectl exec <RayCluster name>-head-svc "ray status"
```

Comment:
To exec commands, you need supply the name of a pod. Instructions for this bit go
 
First identify the name of the head pod  
```kubectl get pod | grep <RayCluster name>-head```

Then run

```
kubectl exec <head pod> -- ray status
```